src/lib/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { clsx, type ClassValue } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+export function cn(...inputs: ClassValue[]) {
+  return twMerge(clsx(inputs));
+}
+
+export function buildGitHubSearchUrl({
```

Comment:
Consider moving this function to the `useRepoSearch` file, as it's unlikely you'll need it anywhere else other than for calling the search repos endpoint.